oci/oci-explanation/ubuntu-oci-container.rst

```diff
@@ -0,0 +1,22 @@
+.. _ubuntu-oci-container-images:
+
+Ubuntu OCI container images
+===========================
+
+The Open Container Initiative (OCI) establishes standards for constructing container 
+images that can be reliably installed across a variety of compliant host environments.
+
+Ubuntu’s `LTS Docker Image Portfolio <https://ubuntu.com/security/docker-images>`_ 
+provides OCI-compliant images that receive stable security updates and predictable 
+software updates, thus ensuring consistency in both maintenance schedule and operational 
+interfaces for the underlying software your software builds on.
+
+Moreover, `Ubuntu-based containers <https://ubuntu.com/containers>`_ (like
+chiselled container images) all leverage the Ubuntu container image as their
+starting point, and thus also profit from its support and security commitments.
```

Comment:
```suggestion
starting point, and so also benefit from its support and security commitments.
```

Reply:
Also, is there a page (from security, perhaps) that outlines the security commitments?

Actually now that I think about it, we might also want to find a different word than "support", since that has connotations of paid support.

Reply:
Y, adding a link to https://ubuntu.com/security/docker-images

Wrt the "support" word, I get what you're saying. In reality there's a notion of support for LTS too - Long Term Support - which is freemium and that's what we try to convey here.